docs/source/tutorials/datalab.ipynb

```diff
@@ -0,0 +1,1393 @@
+       "        'low', 'mid', 'low', 'low', 'high', 'mid', 'low', 'mid', 'mid',\n",
+       "        'mid', 'mid', 'low', 'mid', 'mid', 'mid', 'high', 'mid', 'mid',\n",
+       "        'low', 'mid', 'mid', 'high', 'mid', 'mid', 'mid', 'mid', 'mid',\n",
+       "        'high', 'low', 'mid', 'mid', 'mid', 'mid', 'mid', 'low', 'mid',\n",
+       "        'mid', 'mid', 'mid', 'low', 'mid', 'low', 'mid', 'low', 'mid',\n",
+       "        'mid', 'high', 'low', 'mid', 'mid', 'mid', 'high', 'mid', 'mid',\n",
+       "        'mid', 'high', 'mid', 'mid', 'mid', 'mid', 'mid', 'high', 'mid',\n",
+       "        'mid', 'mid', 'mid', 'mid', 'mid', 'high', 'mid', 'mid', 'low',\n",
+       "        'mid', 'high', 'low', 'high', 'high', 'mid'], dtype='<U4'),\n",
+       " 'confident_joint': array([[23,  0,  6],\n",
+       "        [ 0, 26,  2],\n",
+       "        [ 1,  3, 71]]),\n",
+       " 'classes_by_label_quality':   Class Name  Class Index  Label Issues  Inverse Label Issues  Label Noise  \\\n",
+       " 0       high            0             6                     1     0.206897   \n",
+       " 1        low            1             2                     3     0.071429   \n",
+       " 2        mid            2             4                     8     0.053333   \n",
+       " \n",
+       "    Inverse Label Noise  Label Quality Score  \n",
+       " 0             0.041667             0.793103  \n",
+       " 1             0.103448             0.928571  \n",
+       " 2             0.101266             0.946667  ,\n",
+       " 'overlapping_classes':   Class Name A Class Name B  Class Index A  Class Index B  \\\n",
+       " 0         high          mid              0              2   \n",
+       " 1          low          mid              1              2   \n",
+       " 2         high          low              0              1   \n",
+       " \n",
+       "    Num Overlapping Examples  Joint Probability  \n",
+       " 0                         7           0.053030  \n",
+       " 1                         5           0.037879  \n",
+       " 2                         0           0.000000  ,\n",
+       " 'py': array([0.18181818, 0.21969697, 0.59848485]),\n",
+       " 'noise_matrix': array([[0.95833333, 0.        , 0.07594937],\n",
+       "        [0.        , 0.89655172, 0.02531646],\n",
+       "        [0.04166667, 0.10344828, 0.89873418]]),\n",
+       " 'inverse_noise_matrix': array([[0.79310345, 0.        , 0.01333333],\n",
+       "        [0.        , 0.92857143, 0.04      ],\n",
+       "        [0.20689655, 0.07142857, 0.94666667]]),\n",
+       " 'class_names': ['high', 'low', 'mid']}"
+      ]
+     },
+     "execution_count": 16,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "lab.get_info(\"label\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 6. Get more information from Datalab attributes"
```

Comment:
Let's delete this final section (prefer folks use the helper methods than the attributes directly). But keep the final line with the links to advanced tutorial and examples

Reply:
clarify the caution about info being a messy huge dict, next to where you present the `get_info()` method.